apps/main-landing/src/components/hero-section/components/background-lines.tsx

```diff
@@ -0,0 +1,14 @@
+import Image from 'next/image';
+
+export const BackgroundLines = () => {
+  return (
+    <Image
+      priority
+      width={1820}
+      height={3702.656}
+      src="/hero-background-lines.png"
+      className="absolute top-0 hidden opacity-40 lg:block"
+      alt=""
+    />
+  );
+};
```

Comment:
I solved issue of type-check in CI so you can move back to locally imported images, by doing so you don't need to apply width and height props since Nextjs detects them automatically

Reply:
also given this you probably don't need separate component as it's redundant abstraction, doing this feels good enough, no need to abstract to separate component in my opinion:
```ts
    <Image
      priority
      src={backgroundLines}
      className="absolute top-0 hidden opacity-40 lg:block"
      alt=""
    />
```